proto/buf/alpha/registry/v1alpha1/doc.proto

```diff
@@ -387,6 +389,6 @@ message FileExtension {
   Location location = 4;
   // fields are all the fields that are associated with the extension.
   repeated Field fields = 5;
-  // file_deprecated is the deprecation status of the file containing the field.
-  bool file_deprecated = 6;
+  // implicitly_deprecated is true if its enclosing file or parent element is deprecated.
```

Comment:
In this case, I don't think extensions can have parent elements so I'd tweak the comment a bit here.

Reply:
They can:

```proto
syntax = "proto3";
option (Foo.custom_file_option) = true;
import "google/protobuf/descriptor.proto";
message Foo {
    extend google.protobuf.FileOptions {
        bool custom_file_option = 123456;
    }
}
```

Reply:
😬 ah good catch... Thanks Timo.